src/Microsoft.SqlTools.ServiceLayer/Connection/ConnectionService.cs

```diff
@@ -1142,6 +1142,11 @@ public static SqlConnectionStringBuilder CreateConnectionStringBuilder(Connectio
                         throw new ArgumentException(SR.ConnectionServiceConnStringInvalidAuthType(connectionDetails.AuthenticationType));
                 }
             }
+            if (!string.IsNullOrEmpty(connectionDetails.ColumnEncryptionSetting))
+            {
+                connectionBuilder.ColumnEncryptionSetting =  connectionDetails.ColumnEncryptionSetting == "Enabled" 
+                ? SqlConnectionColumnEncryptionSetting.Enabled : SqlConnectionColumnEncryptionSetting.Disabled;

```

Comment:
you can use enum parser to covert string to enum

Reply:
And make sure you're handling case sensitivity properly - i.e. both enabled and Enabled I would expect to be valid

Reply:
I thought about doing this at first, but wanted to make sure that we are not throwing an `ArgumentException` in the case that we get a value that is a name, but not one of the named constants defined for the enumeration. Since I wasn't certain that the value was guaranteed to be one of the named constants defined for the enumeration, I decided to go with a safe option to enable when the value matched and disable in all other cases. I can certainly use the `Enum.Parse()` method if this is the preferred way.

Reply:
I changed the way this was assigned so that it is not case sensitive and so that an invalid value throws the appropriate exception